src/AWS/Orleans.Clustering.DynamoDB/Orleans.Clustering.DynamoDB.csproj

```diff
@@ -22,6 +22,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\Orleans.Core.Legacy\Orleans.Core.Legacy.csproj" />
```

Comment:
I've pulled all references of Cluster config out of the core packages, but not all of the extension libraries.  So any library that still references cluster config needs depend on core legacy.  This can (and IMHO should) be handled in smaller, more targeted change sets rather than adding to this one.